src/DevCenter/DevCenterData.AutoRest/custom/Deploy-AzDevCenterUserEnvironment.ps1

```diff
@@ -53,172 +55,177 @@ BODY <IEnvironment>: Properties of an environment.
 INPUTOBJECT <IDevCenterdataIdentity>: Identity Parameter
   [Name <String>]: The name of an action that will take place on a Dev Box.
   [CatalogName <String>]: The name of the catalog
+  [CustomizationGroupName <String>]: A customization group name.
+  [CustomizationTaskId <String>]: A customization task ID.
   [DefinitionName <String>]: The name of the environment definition
   [DevBoxName <String>]: The name of a Dev Box.
   [EnvironmentName <String>]: The name of the environment.
   [Id <String>]: Resource identity path
+  [OperationId <String>]: The id of the operation on a Dev Box.
   [PoolName <String>]: The name of a pool of Dev Boxes.
   [ProjectName <String>]: The DevCenter Project upon which to execute operations.
   [ScheduleName <String>]: The name of a schedule.
+  [TaskName <String>]: A customization task name.
   [UserId <String>]: The AAD object id of the user. If value is 'me', the identity is taken from the authentication context.
 .Link
 https://learn.microsoft.com/powershell/module/az.devcenter/deploy-azdevcenteruserenvironment
 #>
 function Deploy-AzDevCenterUserEnvironment {
-  [OutputType([System.Boolean])]
-  [CmdletBinding(DefaultParameterSetName = 'CreateExpanded', PositionalBinding = $false, SupportsShouldProcess, ConfirmImpact = 'Medium')]
+  [OutputType([Microsoft.Azure.PowerShell.Cmdlets.DevCenterdata.Models.Api20240501Preview.IEnvironment])]
```

Comment:
The output type will be updated from bool to IEnvironment. This is also considered as breaking change but I did not see it in the breaking change announcement. May I know if it would cause break to the customers?